src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -2,16 +2,68 @@ import React, { Component } from 'react';
 import Map from '../../Map/Map.js';
 import { AccountType } from '../../Enums';
 import { accountsRef, donatingAgenciesRef } from '../../FirebaseConfig.js';
+import moment from 'moment-timezone';
 
 class OrganizationDetails extends Component {
 
     constructor(props) {
         super(props);
         this.state = {
-            isEditing: false
+            isEditing: false,
+            times: [],
         };
+        this.convertTimes = this.convertTimes.bind(this);
+        this.dayRow = this.dayRow.bind(this);
+        this.dayNames = ['Sun','Mon','Tue','Wed','Thur','Fri','Sat'];
+
+
+    }
+
+    dayRow(i, day) {
+
+        var checkboxName = day + 'Check';
```

Comment:
You should always use "let" when declaring variables in JS. Using var has some weird scoping stuff, here's an explanation: https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable-in-jav